script/configs/holesky/M2_deploy_from_scratch.holesky.config.json

```diff
@@ -0,0 +1,58 @@
+{
+  "chainInfo": {
+    "chainId": 17000
+  },
+  "multisig_addresses": {
+    "pauserMultisig": "0x53410249ec7d3a3F9F1ba3912D50D6A3Df6d10A7",
+    "communityMultisig": "0xCb8d2f9e55Bc7B1FA9d089f9aC80C583D2BDD5F7",
+    "operationsMultisig": "0xfaEF7338b7490b9E272d80A1a39f4657cAf2b97d",
+    "executorMultisig": "0x28Ade60640fdBDb2609D8d8734D1b5cBeFc0C348",
+    "timelock": "0xcF19CE0561052a7A7Ff21156730285997B350A7D"
```

Comment:
looks like this contract isn't verified in the explorer?
https://holesky.etherscan.io/address/0xcF19CE0561052a7A7Ff21156730285997B350A7D#code
just noticing this; would be good to take care of so it's not forgotten

Reply:
https://holesky.etherscan.io/address/0xcF19CE0561052a7A7Ff21156730285997B350A7D#code
verified now